src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestBase.cs

```diff
@@ -36,6 +36,11 @@ public static WasmHost RunningOn
             => WasmHost.Chrome;
 #else
             => WasmHost.Firefox;
+#endif
+#if WASM_ENABLE_THREADS
+        public static bool WasmEnableThreads = true;
```

Comment:
I would suggest an environment variable like `WASM_TESTS_USING_VARIANT=..` which have a default empty value, and can optionally be `=multithreaded` to run with the mt variant of the runtime.